example-android-kts/app/build.gradle.kts

```diff
@@ -30,24 +29,13 @@ android {
     sourceSets {
         getByName("main").java.srcDirs("src/main/kotlin")
     }
-
-    lintOptions {
-        setAbortOnError(false)
-    }
```

Comment:
No longer needed?

Reply:
Not idea honestly, Gradle kept complaining that it wasn't a valid function anymore and I didn't want to sped the time to fix it 🤣

Reply:
:P ok!